internal/http/services/owncloud/ocs/data/capabilities.go

```diff
@@ -172,6 +172,7 @@ type CapabilitiesFilesSharingPublic struct {
 	Password           *CapabilitiesFilesSharingPublicPassword   `json:"password" xml:"password"`
 	ExpireDate         *CapabilitiesFilesSharingPublicExpireDate `json:"expire_date" xml:"expire_date" mapstructure:"expire_date"`
 	CanEdit            ocsBool                                   `json:"can_edit" xml:"can_edit" mapstructure:"can_edit"`
+	Alias              ocsBool                                   `json:"alias" xml:"alias"`
```

Comment:
Naming can be discussed - Suggestions welcome!

Reply:
See also https://github.com/owncloud/ocis/pull/3991#pullrequestreview-1012158015